packages/backend/src/indexers/CacheInvalidationIndexer.test.ts

```diff
@@ -0,0 +1,84 @@
+import { Logger } from '@l2beat/backend-tools'
+import { ChainId, Hash256, UnixTime } from '@lz/libs'
+import { expect, mockFn, mockObject } from 'earl'
+
+import { BlockNumberRepository } from '../peripherals/database/BlockNumberRepository'
+import { IndexerStateRepository } from '../peripherals/database/IndexerStateRepository'
+import { ProviderCacheRepository } from '../peripherals/database/ProviderCacheRepository'
+import { BlockNumberIndexer } from './BlockNumberIndexer'
+import { CacheInvalidationIndexer } from './CacheInvalidationIndexer'
+
+describe(CacheInvalidationIndexer.name, () => {
+  describe(CacheInvalidationIndexer.prototype.invalidate.name, () => {
+    it('should skip initial invalidation', async () => {
```

Comment:
Is it because we do not want to delete the cache on the first startup of the app?

Reply:
Yeah, but given that discovery has run at least once, maybe we should not do that?
I mean isn't that the case only when we have cache preloaded i.e. from the production and rest of the tables? (notably `indexer_states` are empty) - thus they will invalidate to the 0 effectively pruning cache